pook/api.py

```diff
@@ -1,14 +1,23 @@
-import re
 import functools
-from inspect import isfunction
+import re
 from contextlib import contextmanager
-from .engine import Engine
+from inspect import isfunction
 
+from .engine import Engine
+from .matcher import MatcherEngine
 from .mock import Mock
+from .mock_engine import MockEngine
 from .request import Request
 from .response import Response
-from .matcher import MatcherEngine
-from .mock_engine import MockEngine
+
+try:
+    from asyncio import iscoroutinefunction
+except ImportError:
+    iscoroutinefunction = None
+if iscoroutinefunction is not None:
+    from .activate_async import activate_async
+else:
+    activate_async = None
```

Comment:
What about `activate_async = lambda x: False`, and then simplify `if` statement with:
```python
if iscoroutinefunction(fn):
   return activate_async(fn, _engine)
```

Reply:
no because of
line 17 and 18
```
if iscoroutinefunction is not None:
    from .activate_async import activate_async
```

Reply:
Right, thanks!